xbmc/guilib/GUITextLayout.h

```diff
@@ -139,11 +150,13 @@ class CGUITextLayout
   CGUIFont *m_borderFont;  // only used for outlined text
   CGUIFont* m_monoFont = nullptr; //!< Mono-space font to use
   CGUIFont* m_varFont;    //!< Varible-space font to use
-
   bool  m_wrap;            // wrapping (true if justify is enabled!)
   float m_maxHeight;
-  // the default color (may differ from the font objects defaults)
-  UTILS::Color m_textColor;
+
+  // Colour information
+  UTILS::Color m_textColor; // the default color (may differ from the font objects defaults)
+  UTILS::Color m_bgcolor; // background colour to use below the font
```

Comment:
this is not used by the class. if you add this here, you should render the background by this class too